Back/Reciplastk.Gateway/Services/ProductsService.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Azure;
+using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+
+namespace Reciplastk.Gateway.Services
+{
+    public class ProductsService
+    {
+
+        private readonly ReciplastkContext db;
+
+        public ProductsService( ReciplastkContext dbReciplastk)
+        {
+            this.db = dbReciplastk;
+        }
+        private Product GetProductbyId(int id)
+        {
+            var product = db.Products.Where(p => p.Productid == id).FirstOrDefault();
```

Comment:
Please start using Soft delete logic.
```suggestion
            var product = db.Products.Where(p => p.Productid == id && p.IsActive).FirstOrDefault();
```